synapse/cortex.py

```diff
@@ -434,6 +437,27 @@ def _convert_reqdict(reqdict):
                 await node.set(name, valu)
                 return node.pack()
 
+    async def delNodeProp(self, iden, name):
+        '''
+        Delete a property from a single node.
+        '''
+
+        buid = s_common.uhex(iden)
+
+        async with await self.cell.snap(user=self.user) as snap:
+
+            with s_provenance.claim('coreapi', meth='prop:del', user=snap.user.iden):
+
+                node = await snap.getNodeByBuid(buid)
+                if node is None:
+                    raise s_exc.NoSuchIden(iden=iden)
+
+                prop = node.form.props.get(name)
+                await self._reqUserAllowed('prop:del', prop.full)
+
+                await node.pop(name)
```

Comment:
If this returns False, the operation failed.  Do we want to push that up to a remote caller, or allow them to figure out from the packed node they get returned?

Reply:
I think we just let that go by.  Ultimately, if somebody else deleted the property between when you got the node and when you deleted the prop, it's all good :)